### ISSUE TITLE
Existing resolutions

### ISSUE THREAD

**Issue by evanp** [swicg/potential-charters#9]
We have a number of resolutions made over the years visible at https://github.com/swicg/meetings/blob/main/RESOLUTIONS.md .

For the CG charter, we should either make sure the charter is compatible with those resolutions, or make sure that we explicitly obsolete them.

**Comment by bumblefudge** [comment]
I checked over them and the only two possible issues I see _so far_ are that:
1. the discussions on [2024-08-02](https://github.com/swicg/meetings/tree/main/2024-08-02#process-question--needs-fep-handling) about using "FEP needed" might align perfectly with [the line in the new cg/wg document](https://github.com/swicg/potential-charters/pull/4/commits/e4fac555ec021ff5921a74f8ba749372d53acd70#diff-d81012efbd769bfcf839266111e957683692c0665a78aeb25cf5b5370440e6c6R32) about CG "requesting" FEP before leaving Stage 0
2. I have no idea what happened to the Policy Group from 2021? Do Task Forces "stale out"?

Oh, that reminds me, we should probably PR in a definition of what a Task Force is, even if it's just a one-sentence reference to the W3C-wide definition of them, since we're already doing so many of them 😅

**Comment by gobengo** [comment]
> we should either make sure the charter is compatible with those resolutions, or make sure that we explicitly obsolete them.

Can you explain why the charter should do that? I hear the recommendation, but don't see the explanation. Without an explanation, I'd say let's not block on this if we don't have to.

If the concern is to avoid ambiguity about incompatibilities, I can see that. I'd assume charter in general would always trump resolutions. At the same time, I don't expect there will be many conflicts.

IMO let's do nothing or include one sentence "If this charter conflicts with any resolutions, the charter takes precedence. The group should create a report for each conflict."

Is there a specific resolution you worry would be in conflict with a specific potential charter?

**Comment by bumblefudge** [comment]
I agree with Bengo that none of these should block the CG Charter being merged, although I also think they're all worth of, say, separate mailing list threads or issues on respective repos?

> Going through the [resolutions](https://github.com/swicg/meetings/blob/main/RESOLUTIONS.md) in reverse order, here are some that seem relevant:
> 
> * _Accept majority of straw poll as SocialCG for group shortname._

Hehe, there's a separate issue for this, but i'm not sure it should block Charter either way.  (I'm #teamSocialCG for the record, just not a hill I'll die on).

> * _Use https://www.w3.org/wiki/ActivityPub_extensions as the directory for extensions that links to them wherever authors want to write them up (github repo, wiki, personal site), and then use github issues for issue discussions._
> * _Create an activitypub-extensions repo under the SWICG GitHub org in order to discuss AP extensions. Move existing "postponed (revisit in future effort)" discussions there. Create a separate repository if we want to actually write out some spec text for an extension._
> * _RESOLVED (pending acceptance from jasnell): Discuss and curate ActivityStreams extensions in the ActivityStreams git(hub) repository, with normal issues / PRs, etc_
> * _The CG will maintain an editors' draft of the AS2 document, applying the editorial changes listed in the ERRATA document_

Can we figure out how to formalize these during or after WG charter discussions? This feels like WG scope that CG was holding for safe-keeping, not sure if the CG needs to change or if the status of this work needs to change before CG charter can be ratified.

> * _The SocialCG should support the FEP (Fediverse Enhancement Proposal) process as a way of documenting extensions to the fediverse and encourage bringing FEP proposals for discussion to the SocialCG._

Current staging process already does that? Issue #33 is already open if people have more specific requests or PRs that would close it.

> * _the remit of the Policy Group is primarily to liase with existing digital/tech political lobbying groups to bring awareness of relevant standards and W3C recommendations which can then be relayed at the right times and in the right places to politicians and policy makers. As part of this, the Policy Group may organise events, briefings, draft documentation, proposals, or open letters which can be co-signed by the SocialCG._

Ooh, good point, we never heard from this "Task Force" again.  Is it still active? Does someone know Sebastian's github handle, or that of anyone else who might want to "reboot" the effort as a Task Force ?

> * _SocialCG starts up a Website Taskforce to update activitypub.rocks and as2.rocks (only). Bandwidth permitting. And set up a process for future updates._

PR #53 defines Task Forces as driving a deliverable "or some other objective", and I had the Website TF in mind, since "refresh the website" is another objective/target-state. @jernst is there anything about the draft CG charter or draft staging process that doesn't work for the website TF?  Anything else that needs to be done now/before CG chartering?

**Comment by evanp** [comment]
Maybe we could add a section like this:

> # Previous decisions
>
> From 2017 to 2024 the SocialCG operated under the same process as its predecessor, the Social Web Working Group.
> The terms of this charter supercede any decisions made during that time that contradict the terms of the charter. Any 
> compatible decisions stand.

**Comment by evanp** [comment]
I added this text to a PR.

**Comment by dmitrizagidulin** [comment]
Thanks again to @evanp for creating PR #59. Merged, closing this issue.
